types/rusha/index.d.ts

```diff
@@ -4,17 +4,15 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.0
 
-/// <reference types="node" />
-
 interface Hash {
-    update(value: string|number[]|ArrayBuffer|Buffer): Hash;
+    update(value: string|number[]|ArrayBuffer): Hash;
     digest(encoding?: undefined): ArrayBuffer;
     digest(encoding: 'hex'): string;
 }
 
 interface RushaWorkerRequest {
     id: string;
-    data: string|number[]|ArrayBuffer|Buffer|Blob;
+    data: string|number[]|ArrayBuffer|Blob;
```

Comment:
This one I'm not sure about, as there may be other restrictions on the type based on web workers and `postMessage` behavior.

```suggestion
    data: string | number[] | ArrayBuffer | { buffer: ArrayBuffer } | Blob;
```